src/components/Auth/AUTHCallback.js

```diff
@@ -7,21 +7,21 @@ const AUTHCallback = () => {
 
   useEffect(() => {
     handleAuthentication({ postLoginRoute: "/feed" });
-  }, []);
+  }, [handleAuthentication]);
```

Comment:
nooooo this auto linting. we don't want anything inside the dependencies.

im going to add this to lint ignore

Reply:
Could you undo it